File: samples_generated_code/samples_1/qwen/baseline_code_java/code_row_107.java
public class Main {
    public static void main(String[] args) {
        Person john = allocatePerson();
        john.status = 0;
        john.name = "John";
        System.out.println("Name: " + john.name + ", Status: " + john.status);
    }

    private static Person allocatePerson() {
        return new Person();
    }
}

class Person {
    int status;
    String name;
}

File: samples_generated_code/sample_3/athene/baseline_code_c/code_row_107.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

typedef struct {
    char* name;
    int status;
} Person;

int main() {
    Person *p = (Person*)malloc(sizeof(Person));
    if (p == NULL) return 1; // Check for allocation failure

    p->status = 0;
    p->name = strdup("John"); // Duplicate the string to allocate memory for name

    // Free allocated memory
    free(p->name);
    free(p);

    return 0;
}